### PR TITLE
inversion count with fenwick tree

### PR DESCRIPTION
**Pull request by ronistone**
Changes: Add Counting inversion with Fenwick Tree

### CODE REVIEW

File: code/data_structures/fenwick_tree/fenwick_tree_inversion_count.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+{
+	register char c = getchar_unlocked();
+	*x = 0;
+	for(; (c<48)||(c>57);c = getchar_unlocked());
+	for(; (c>47)&&(c<58);c = getchar_unlocked())
+		*x = (int)((((*x)<<1) + ((*x)<<3)) + c - 48);
+}
+
+
+main(){
```
Comment (arnavb) [inline code/data_structures/fenwick_tree/fenwick_tree_inversion_count.cpp]: main should have a return type, especially in C++.

**Comment by ronistone** [comment]
sorry, fixed.

**Comment by arnavb** [comment]
Also, just on a related note, since this is C++, you may consider using `vector`s, instead of raw arrays, since they are easier to work with and have more flexibility.

**Comment by ronistone** [comment]
OK, I used int because of time.
I updated to use vector

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Comment by arnavb** [comment]
@ronistone Where have you used a vector 😐 ?

**Pull request merged**